lib/hammox.ex

```diff
@@ -333,16 +342,20 @@ defmodule Hammox do
   end
 
   defp fetch_typespecs(behaviour_module_name, function_name, arity) do
-    {:ok, callbacks} = Code.Typespec.fetch_callbacks(behaviour_module_name)
-
-    callbacks
-    |> Enum.find_value([], fn
-      {{^function_name, ^arity}, typespecs} -> typespecs
-      _ -> false
-    end)
-    |> Enum.map(fn typespec ->
-      Utils.replace_user_types(typespec, behaviour_module_name)
-    end)
+    case Code.Typespec.fetch_callbacks(behaviour_module_name) do
+      {:ok, callbacks} ->
+        callbacks
+        |> Enum.find_value([], fn
+          {{^function_name, ^arity}, typespecs} -> typespecs
+          _ -> false
+        end)
+        |> Enum.map(fn typespec ->
+          Utils.replace_user_types(typespec, behaviour_module_name)
+        end)
+
+      :error ->
```

Comment:
What is the reason behind handling `:error` here? I don't see a use case for this function to silently proceed when the `Code.Typespec.fetch_callbacks(behaviour_module_name)` call errors — I'd rather make it fail to be stricter.

Reply:
Actually, looks like the `Code.Typespec.fetch_callbacks(behaviour_module_name)` calls error only then the module doesn't exist. So I think there is a need to add validation of behavior module inside `protect/2` function and raise an exception.